tools/wpt/testfiles.py

```diff
@@ -252,6 +260,15 @@ def affected_by_wdspec(test):
                     break
         return affected
 
+    def affected_by_interfaces(file_contents):
+        if len(interfaces_changed_names) > 0:
+            if 'idlharness.js' in file_contents:
+                for interface in interfaces_changed_names:
```

Comment:
Old comment was resolved by changing indent, so again: This will work for `idl_test`, but tests still fetching /interfaces/something.idl would not be considered affected. Maybe also look for `interface + '.idl'`?